inc/Engine/Optimization/RUCSS/Admin/Subscriber.php

```diff
@@ -66,9 +67,34 @@ public static function get_subscribed_events() : array {
 			'admin_post_rocket_clear_usedcss'    => 'truncate_used_css_handler',
 			'admin_notices'                      => 'clear_usedcss_result',
 			'rocket_admin_bar_items'             => 'add_clean_used_css_menu_item',
+			'rocket_after_settings_checkbox'     => 'display_progress_bar',
+			'admin_print_styles-settings_page_' . $plugin_slug => 'add_admin_js',
```

Comment:
```suggestion
			'admin_enqueue_scripts" => 'add_admin_js',
```

And do the `hook_suffix` check inside the method itself